playbook/Gemfile.lock

```diff
@@ -2,9 +2,9 @@ PATH
   remote: .
   specs:
     playbook_ui (10.6.0)
-      actionpack (>= 5.2.4.5, < 7.0)
-      actionview (>= 5.2.4.5, < 7.0)
-      activesupport (>= 5.2.4.5, < 7.0)
+      actionpack (>= 5.2.4.5, < 6.0)
+      actionview (>= 5.2.4.5, < 6.0)
+      activesupport (>= 5.2.4.5, < 6.0)
```

Comment:
Running `bundle update` before this change updates to `rails 6.0`! We are not ready for this yet.